prov/verbs/src/verbs_info.c

```diff
@@ -1138,6 +1137,7 @@ static int fi_ibv_set_info_addrs(struct fi_info *info,
 				if (ret)
 					return ret;
 			}
+			iter_info->addr_format = FI_ADDR_IB_UD;
```

Comment:
Verified that this single line change alone is sufficient to resolve the fabtests issue.